docs/pages/components/localization-editor.md

```diff
@@ -22,31 +22,31 @@ summary:
             <input type="text" class="" id="" aria-controls="NJvVh542c" aria-expanded="false"
             aria-haspopup="true" placeholder="Enter Label">
             <span class="fd-input-group__addon fd-input-group__addon--after fd-input-group__addon--button">
-            <button class="fd-button--light fd-localization-editor__button"
+            <button class="fd-button--light"
             aria-controls="test" aria-expanded="false" aria-haspopup="true">EN*</button>
             </span>
          </div>
       </div>
       <div class="fd-popover__body fd-popover__body--no-arrow"  aria-hidden="true" id="NJvVh542c">
          <nav class="fd-menu">
-            <ul class="fd-menu__list fd-localization-editor__list">
```

Comment:
This class is functionally being replaced by `fd-localization-editor__language` which will be placed directly on the `<li>` elements to adjust spacing.